scripts/reana

```diff
@@ -104,6 +106,134 @@ down () {
+                    "{\"experiment\":\"$experiment\",
+                      \"toplevel\":\"github:diegodelemos/reana-demo-helloworld@2-add-helloworld-workflow\",
+                      \"workflow\": \"hello.yaml\",
+                      \"nparallel\": \"10\",
+                      \"preset_pars\": {\"name\": \"J.Doe\", \"delay\": \"2\"}
+                    }" \
+                    http://"$(reana get reana-workflow-controller)"/yadage \
+                    | grep -oP '(?<="workflow_id": ")[^"]*')
+
+    # Check for successful execution of curl
+    # https://unix.stackexchange.com/a/204790
+    # Note that checking with '$?' might fail on some corner cases
+    # AND when authentication is involved (response codes 401 and 407)
+    submit_status=$?
+
+    if [ "$submit_status" = "0" ]; then
+
+        if [ "$flag_verbose" != "" ]; then
+            echo "[INFO] Open http://$controller_url/$workflow_id to see visual representation of your workflow and it's progress."
+            echo "[INFO] You can 'cat' or 'tail' /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt -file"
+            echo "       from shared storage of REANA cluster to see how helloworld workflow progresses."
+            echo "[INFO] (e.g. minikube ssh \"tail -c+1 -F /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt\")"
+        fi
+
+        echo ""
+        echo "Workflow_id: $workflow_id"
+        echo "Monitor: http://$controller_url/$workflow_id"
+        echo "Output:  /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt"
+        echo ""
+
+        if [ "$flag_devel" != "" ]; then
+            # Start tailing of job-controller logs
+            #kubectl logs -f $(kubectl get pods | grep -Po 'job-controller[^ ]+')
+
+            # Start tailing of yadage-atlas-worker logs
+            #kubectl logs -f $(kubectl get pods | grep -Po 'yadage-atlas-worker[^ ]+')
+
+            # Open workflow-monitor web-interface
+            # Inspired by http://stackoverflow.com/a/27776822
+            os_type=$(uname -s)
+            if [ "$os_type" = "Darwin" ]; then
+                open "http://$controller_url/$workflow_id"
+            elif [ "$os_type" = "Linux" ] ; then
+                xdg-open "http://$controller_url/$workflow_id"
+            fi
+
+            # Tail output-file of the workflow
+            echo "[INFO] Tailing outputfile...hit Ctrl+C when done."
+            echo ""
+            minikube ssh "tail -c+1 -F /reana/$experiment/$workflow_id/yadage/helloworld/greetings.txt"
+        fi
+
+        retval=0
+    else
+        retval=1
+    fi
+
+    return $retval
+}
+
+run () {
+    # Used to start workflows on REANA cluster
+    # Currently hardcoded to run REANA helloworld demo workflow (L210).
+    # (L204) Experiment hardcoded to 'atlas' for now because of
+    # reanahub/reana-workflow-engine-yadage/issues/20
+
+    if [ "$3" = "-e" ]; then
+        experiment=$4
+    else
+        experiment=${REANA_EXPERIMENT-}
```

Comment:
Quote from some SO-post:
_"${var:-} and ${var-} mean different things. ${var:-} will expand to empty string if var unset or null and ${var-} will expand to empty string only if unset"_
BUT now that I think of it, it might be better with ${REANA_EXPERIMENT:-} so it will default to empty string.